README.md

```diff
@@ -97,7 +94,7 @@ const MyConnectedComponent = connect(mapStateToProps, mapDispatchToProps, MyComp
 </BarLeft>
 ```
 
-Instead, you can do something like this:
+Instead, you can do something like this (for Context you can pass the values as prop):
```

Comment:
Or you can use https://github.com/cozy/cozy-ui/blob/master/react/BarContextProvider/index.jsx ? ;) ( cc @JF-Cozy )